docs/python/README.rst

```diff
@@ -8,6 +8,11 @@ For more information on ONNX Runtime, please see `aka.ms/onnxruntime <https://ak
 Changes
 -------
 
+1.10.0
+^^^^^
+
+Release Notes : https://github.com/Microsoft/onnxruntime/releases/tag/v1.10.0
```

Comment:
Shall we add release notes right before 1.10 release?

Reply:
removed